src/core/config.go

```diff
@@ -501,6 +501,10 @@ type Configuration struct {
 	PleaseLocation string
 	// buildEnvStored is a cached form of BuildEnv.
 	buildEnvStored *storedBuildEnv
+
+	FeatureFlags struct {
+		JavaBinaryExecutableByDefault bool `help:"Makes java_binary rules self executable by default. Target release version 16." var:"FF_JAVA_SELF_EXEC"`
+	}
```

Comment:
you should add some help on the struct too so that `plz halp featureflags` has a bit of explanation.

Might be worth mentioning that these generally won't remain around very long relative to other config options (i.e. they're typically enabling "preview" features for the next major).